components/mpas-ocean/src/Registry.xml

```diff
@@ -295,8 +295,8 @@
 					possible_values=".true. or .false."
 		/>
 		<nml_option name="config_Redi_closure" type="character" default_value="constant" units="unitless"
-					description="Control what type of function is used for Redi $\kappa$."
-					possible_values="'constant', 'data'"
+					description="Control what type of function is used for Redi $\kappa$. For 'equalGM', RediKappa is set to gmBolusKappa, so picks up the closure used by GM."
+					possible_values="'constant', 'equalGM', 'data'"
```

Comment:
would it be good to say this only works for 2-d schemes (e.g. constant or visbeck)?  This won't work for edenGreatbatch

Reply:
OK, I added that comment above.